### DISCUSSION TITLE
Series is not displayed

### DISCUSSION THREAD

**Discussion by elevado** [forem/forem#19433] [Q&A]
Hi,

I'm trying to create a series on dev.to. Unfortunately, the series is not displayed. Do I have to position a placeholder somewhere?

// Artikel 1
```
---
series: Name of series
title: Artikel 1
---
```

// Artikel 2
```
---
series: Name of series
title: Artikel 2
---
```

**Comment by thomasbnt** [comment]
Hello @elevado ! If your serie have only one post published, the serie doesn't show to the top and bottom of the content.
